rest-api-spec/src/main/resources/rest-api-spec/test/search/230_interval_query.yml

```diff
@@ -0,0 +1,58 @@
+setup:
+  - skip:
```

Comment:
Is this working ? I was not aware that the `skip` section can be set in the `setup` part but if it's working... ;)